package.json

```diff
@@ -20,7 +20,7 @@
     "@babel/preset-react": "7.12.13",
     "@fortawesome/fontawesome-free": "5.15.2",
     "acorn": "8.0.5",
-    "adhocracy4": "liqd/adhocracy4#0a066f3be27d61b641c6260a8eb248e4d12b61d3",
+    "adhocracy4": "liqd/adhocracy4#cf583fde42ac6aae4eaafdab7b338b4d83cea9bd",
```

Comment:
Ah, that's where the wrong versions came from yesterday. :) We left the hash out of depending PRs and only added it after the a4 PRs are merged. Then, people had to locally try it by either adding the a4 hash or using the local a4 and there are no wrong hashes introduced by accident.